napari/_vispy/overlays/scale_bar.py

```diff
@@ -176,7 +176,7 @@ def _on_text_change(self):
         # update the dpi scale factor to account for screen dpi
         # because vispy scales pixel height of text by screen dpi
         if self.node.text.transforms.dpi:
-            # use 96 as the reference dpi
+            # use 96 as the napari reference dpi for historical reasons
```

Comment:
96 is the hardcoded logical ([it's complicated...](https://en.wikipedia.org/wiki/Dots_per_inch#Computer_monitor_DPI_standards)) dpi of the majority monitors (except macs), so it's not quite historical reasons but practical :P